baybe/searchspace/continuous.py

```diff
@@ -69,20 +70,20 @@ def from_bounds(cls, bounds: pd.DataFrame) -> SubspaceContinuous:
         return SubspaceContinuous(parameters)
 
     @classmethod
-    def from_dataframe(cls, df: pd.DataFrame) -> SubspaceContinuous:
+    def from_dataframe(cls, dataframe: pd.DataFrame) -> SubspaceContinuous:
```

Comment:
`dataframe` is just a long version of the very common `df`, why rename? thats prob the most commonly used convention in all of data science python :O

Reply:
I know 😃 And we can also stick with `df` if you prefer, depending on what you think about that one issue that you might not have considered: in the current setting, the name of the parameter is what you need to provide in your config, i.e. you'd need to write
```json
"constructor": "from_dataframe",
"df": ...
```
That was my reason for the change. If you think that is better than the current version, I'm happy to undo

Reply:
ok didnt thing about that, true
but in my midn `df` is still as clear as `dataframe` and would break the code consistency? People will have to look up constructor signatures anyway, so why bother with this?

but ill leave it to you, maybe @AVHopp has an opinion, feel free to resolve

Reply:
My feeling is also that this is overthinking. And also, we have type hints for telling people what they actually need to provide, so there is no ambiguity here (assuming that people read the type hints which I think we should assume). So my vote is to go for `df` instead of dataframe but keep the constructor name.